### ISSUE TITLE
Accessibility: "Torrent Option" window doesn't take focus when adding a torrent

### ISSUE THREAD

**Issue by vvillenave** [transmission/transmission#1714]
Open Transmission; hit Ctrl-U to paste a magnet URI then Enter.

The "Torrent Option" window opens _but_ the focus remains in the main window.

The user has to be aware that a new window has actually appeared, which they have to Alt-Tab into before hitting Enter. For visually-impaired users, this is not intuitive.

Ideally, that window should take the focus so that adding a torrent (or magnet link) would be as easy as Ctrl-U then hitting Enter twice (once for the magnet dialog, once for the Torrent Option window).

**Comment by Coeur** [comment]
It works perfectly as expected on macOS, and the shortcut is Cmd+U.
So I'm strongly assuming that the issue is not on macOS client.
Is it on Qt client, GTK client or web client?

**Comment by ckerr** [comment]
I've tested this with both the GTK and Qt clients in 4.1.0 and they both give focus to the Open dialogs. Also, neither of them is named "Torrent Option." So if it's not GTK, Qt, or macOS ... I'm not sure where the issue was?

Closing this ticket as stale due to the inability to reproduce the issue + the lack of response from the reporter
